### PR TITLE
Add spelling corrections for dictionar(y|ies)

### PR DESCRIPTION
**Pull request by cfi-gb**
See e.g.:
- https://github.com/search?q=dictrionary&type=code
- https://grep.app/search?q=dictrionary

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -11159,6 +11159,8 @@ dictionry->dictionary
 dictoinaries->dictionaries
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:11159]:
```suggestion
dictoinaries->dictionaries
dictoinary->dictionary
```

Comment (cfi-gb) [reply codespell_lib/data/dictionary.txt:11159]: Applied :+1:

**Review by vikivivi — APPROVED** [review]
LGTM

**Comment by luzpaz** [comment]
TIA!

**Pull request merged**